matlab/schemas/+stack/Corrections.m

```diff
@@ -0,0 +1,18 @@
+%{
```

Comment:
All these starting with Corrections (Corrections, CorrectionsRaster, CorrectionsMotion, CorrectionsStitched, CorrectionsROICoordinates) are from an older version, they are not in the code or the database anymore